05.wc/wc.rb

```diff
@@ -0,0 +1,70 @@
+# ! /usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+DDISPLAY_ADJUST_LENGTH = 1
```

Comment:
```suggestion
DISPLAY_ADJUST_LENGTH = 1
```

で良かったりしません？

あと、このプログラムには display, print, output という用語が出てきますが、それぞれ意図があって使い分けてますか？（僕からするとどれもほぼ同じ意味に思える）